docs/reference/auditbeat/auditbeat-installation-configuration.md

```diff
@@ -258,6 +258,7 @@ Auditbeat comes with predefined assets for parsing, indexing, and visualizing yo
 
 The default location where Windows log files are stored varies:
 * {applies_to}`stack: ga 9.1` `C:\Program Files\Auditbeat-Data\logs`
```

Comment:
Is this a typo or are there actually 3 different paths? 

```suggestion
* {applies_to}`stack: ga 9.1` `C:\Program Files\Auditbeat-Data\Logs`
```

Reply:
Typo, it should always be lower case.

Well, the actual folder created is lowercase, but Windows file system is case insensitive, so either one works, however it's better to match what our code does.